appveyor.yml

```diff
@@ -21,6 +21,7 @@ install:
 
   # Install the dependencies of the project.
   - pip install numpy scipy matplotlib nose h5py mock
+  - pip install hypothesis
   - pip install pydicom
```

Comment:
Might as well combine all three of these lines. Let pip do the dependency resolution in one shot.